tests/Oryx.BuildImage.Tests/Hugo/HugoDynamicInstallationTest.cs

```diff
@@ -34,6 +34,20 @@ public static TheoryData<string> ImageNameData
             }
         }
 
+        [Fact, Trait("category", "jamstack")]
+        public void PipelineTestInvocationJamstack()
+        {
+            var imageTestHelper = new ImageTestHelper();
+            InstallsHugoVersionDynamically_UsingEnvironmentVariable_AndBuildsApp(imageTestHelper.GetAzureFunctionsJamStackBuildImage());
+        }
+
+        [Fact, Trait("category", "githubactions")]
+        public void PipelineTestInvocationGithubactions()
+        {
+            var imageTestHelper = new ImageTestHelper();
+            InstallsHugoVersionDynamically_UsingEnvironmentVariable_AndBuildsApp(imageTestHelper.GetGitHubActionsBuildImage());
+        }
+
         [Theory]
         [MemberData(nameof(ImageNameData))]
```

Comment:
Are we able to get rid of the above two lines since we won't be running this test without calling it from a wrapping function? We should also be able to make this function private.

There are a couple of spots in the PR where this also occurs, curious if we can make this change throughout the PR.

Reply:
Thanks! I've removed `MemberData` on methods that had the wrapper test method